docs/io-rabbitmq-source.md

```diff
@@ -85,3 +85,104 @@ Before using the RabbitMQ source connector, you need to create a configuration f
       passive: "false"
   ```
 
+## Usage
+
+
+### Standalone mode
+
+This example describes how to use the RabbitMQ source connector to feed data from RabbitMQ and write data to Pulsar topics in the standalone mode.
+
+#### Prerequisites
+
+- There is a RabbitMQ server with some history messages in the queue.
+
+#### Steps
+
+1. Get a Pulsar package and start Pulsar in standalone mode.
+
+   ```bash
+   wget https://archive.apache.org/dist/pulsar/pulsar-@pulsar:version@/apache-pulsar-@pulsar:version@-bin.tar.gz
+   tar xvfz apache-pulsar-@pulsar:version@-bin.tar.gz
+   cd apache-pulsar-@pulsar:version@
+   bin/pulsar standalone
+   ```
+
+2. Download the [nar package](https://archive.apache.org/dist/pulsar/) corresponding to Pulsar's version and copy the following file to Pulsar's directory.
+
+    ```bash
+    wget https://archive.apache.org/dist/pulsar/pulsar-@pulsar:version@/connectors/pulsar-io-rabbitmq-@pulsar:version@.nar
+    cp pulsar-io-rabbitmq-@pulsar:version@.nar ./connectors
+    ```
+
+3. Set the retention of the namespace, otherwise the messages into the Pulsar's topic which have not the subscription have been immediately deleted.
```

Comment:
It's more correct.

```suggestion
3. Messages published to a topic without durable subscriptions are automatically marked as ready for deletion by default. We can set a retention policy at the namespace level to avoid this.
```